Source/Particles/PhysicalParticleContainer.H

```diff
@@ -21,6 +21,9 @@
     #include <QuantumSyncEngineWrapper.H>
     #include <BreitWheelerEngineWrapper.H>
     #include <QedChiFunctions.H>
+
+    #include <QEDPairGeneration.H>
+    #include <QEDPhotonEmission.H>
```

Comment:
```suggestion
    #include "Particles/ElementaryProcess/QEDInternals/QEDPairGeneration.H"
    #include "Particles/ElementaryProcess/QEDInternals/QEDPhotonEmission.H"
```

Reply:
The path is actually just `Particles/ElementaryProcess/` , so I cannot commit directly your suggestion. However I have implemented the change.